lib/promiscuous/config.rb

```diff
@@ -1,21 +1,26 @@
 module Promiscuous::Config
-  mattr_accessor :app, :backend, :amqp_url,
-                 :publisher_amqp_url, :subscriber_amqp_url, :publisher_exchange,
-                 :subscriber_exchanges, :sync_exchange, :queue_name, :queue_options,
-                 :redis_url, :redis_stats_url, :stats_interval, :error_queue_name,
-                 :socket_timeout, :heartbeat, :sync_all_routing, :rabbit_mgmt_url,
-                 :prefetch, :publisher_lock_expiration, :publisher_lock_timeout, :recovery_interval, :logger, :subscriber_threads,
-                 :version_field, :error_notifier, :queue_policy, :test_mode,
-                 :on_stats, :max_retries, :generation, :destroy_timeout, :destroy_check_interval,
+  mattr_accessor :app, :kafka_backend, :kafka_hosts, :zookeeper_hosts, :publisher_topic,
+                 :subscriber_topics, :redis_url, :redis_stats_url, :stats_interval,
+                 :socket_timeout, :heartbeat, :sync_all_routing, :prefetch,
+                 :publisher_lock_expiration, :publisher_lock_timeout,
+                 :recovery_interval, :logger, :subscriber_threads, :version_field,
+                 :error_notifier, :test_mode, :on_stats, :max_retries, :generation,
+                 :destroy_timeout, :destroy_check_interval,
+    # vvvv REMOVE AFTER AMQP IS GONE vvvv
+                 :backend, :amqp_url, :publisher_amqp_url, :subscriber_amqp_url,
+                 :publisher_exchange, :subscriber_exchanges, :sync_exchange, :queue_name,
+                 :queue_options, :error_queue_name, :rabbit_mgmt_url, :queue_policy,
                  :error_exchange, :error_routing, :retry_routing, :error_ttl
 
   def self.backend=(value)
     @@backend = value
     Promiscuous::AMQP.backend = value
+    Promiscuous::Kafka.backend = (value == :bunny)? :poseidon : value
```

Comment:
TODO: refactor to allow for either `bunny` or `kafka` for subscription and both for publishing